Vulkan-Pipeline-Assistant/mainwindow.cpp

```diff
@@ -43,6 +58,26 @@ namespace vpa {
         delete m_ui;
     }
 
+    bool MainWindow::nativeEvent(const QByteArray &eventType, void *message, long *result) {
+        Q_UNUSED(result)
+        Q_UNUSED(eventType)
+        if (QGuiApplication::platformName() == "windows") {
+            MSG* msg = static_cast<MSG*>(message);
+            if (msg->message == WM_ENTERSIZEMOVE) {
+                if (m_vulkan) m_vulkan->Details().window->HandlingResize(true);
+                return true;
+            }
+            else if(msg->message == WM_EXITSIZEMOVE) {
+                if (m_vulkan) {
+                    m_vulkan->Details().window->HandlingResize(false);
+                    if (m_vulkan->State() == VulkanState::Ok) m_vulkan->RecreateSwapchain();
+                }
+                return true;
+            }
+        }
+        return false;
```

Comment:
A lot of indents here.

maybe if not windows return false, at the start
then can also extract return true outside the if and else blocks

Reply:
I've changed this to move the windows specific handling to its own function, to maintain nativeEvent as easily expandable to handle other platforms.